includes/admin/class-wcmp-export.php

```diff
@@ -79,11 +79,15 @@ public function __construct()
      */
     public function exportByOrderId(int $orderId): void
     {
-        $automaticExport = WCMYPA()->setting_collection->isEnabled(WCMYPA_Settings::SETTING_AUTOMATIC_EXPORT);
+        if (! $orderId) {
```

Comment:
Ik zie nu pas dat dit gecheckt wordt, maar ik denk niet dat hij ooit falsy zou moeten zijn. Hij moet altijd een int zijn dus dit is altijd true tenzij er op een of andere manier `0` in komt... En als dat zo is moet dat niet meer kunnen 😬